src/utils/navigation.ts

```diff
@@ -69,6 +69,11 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
     nextPage: []
   },
+  [config.UPDATE_BENEFICIAL_OWNER_TYPE_URL]: {
+    currentPage: config.UPDATE_BENEFICIAL_OWNER_TYPE_PAGE,
+    previousPage: () => config.OVERSEAS_ENTITY_UPDATE_DETAILS_URL,
```

Comment:
should this previous page not also change its next?

Reply:
it should have been OVERSEAS_ENTITY_REVIEW_URL, corrected its next